### DISCUSSION TITLE
How to calculate electrostatic surface?

### DISCUSSION THREAD

**Discussion by sbembenek18** [grimme-lab/xtb#862] [General]
I know how to edit the input file to get a cube file for the electronic density -- but how do I calculate the corresponding electrostatic potential surface?

**Comment by icamps** [comment]
Hopping this can be answer now.
[https://github.com/grimme-lab/xtb/discussions/547#discussion-3732713](https://github.com/grimme-lab/xtb/discussions/547#discussion-3732713)
